downstream/modules/aap-hardening/ref-infrastructure-as-code.adoc

```diff
@@ -23,9 +23,14 @@ The following Ansible content collections are available for managing AAP compone
 | `infra.ee_utilities` | A collection of roles for creating and managing execution environment images, or migrating from the older Tower virtualenvs to execution environments.
 |===
 
-Many organizations use CI/CD platforms to configure pipelines or other methods to manage this type of infrastructure. However, using AAP natively, a webhook can be configured to link a Git-based repository natively. In this way, Ansible can respond to git events to trigger Job Templates directly. This removes the need for external CI components from this overall process and thus reduces the attack surface.
+Many organizations use CI/CD platforms to configure pipelines or other methods to manage this type of infrastructure. However, using {PlatformNameShort} natively, a webhook can be configured to link a Git-based repository natively. In this way, Ansible can respond to git events to trigger Job Templates directly. This removes the need for external CI components from this overall process and thus reduces the attack surface.
 
-Using these practices allows version control of all infrastructure and configuration. Apply git best practices to ensure proper code quality inspection prior to being synchronized into AAP. This includes pull requests, inspection tools to ensure lint rules are adhered to, no plain text secrets are committed to git, pre-commit hooks and any other policies are applied prior to committing.
+These practices allow version control of all infrastructure and configuration. Apply Git best practices to ensure proper code quality inspection prior to being synchronized into {PlatformNameShort}. Relevant Git best practices include the following: 
 
-IaC also encourages using external vault systems (see xref:<link>[<link_External credential vault considerations] which removes the need to store any sensitive data in the repository, or deal with having to individually vault files as needed.
+* Creating pull requests.
+* Ensuring that inspection tools are in place.
+* Ensuring that no plain text secrets are committed.
+* Ensuring that pre-commit hooks and any other policies are followed.
+
+IaC also encourages using external vault systems which removes the need to store any sensitive data in the repository, or deal with having to individually vault files as needed. For more information on using external vault systems, see section link:con-external-credential-vault.adoc[2.3.2.3 External credential vault considerations] within this guide.
```

Comment:
Should this be spelled out? Infrastructure as Code (IaC) or was it previously defined earlier?

Reply:
Yes, it was previously defined in line ten.